test-suite/utils/upm.go

```diff
@@ -138,6 +149,17 @@ func (bt *BackendT) UpmInstall() {
 	}
 }
 
+func normalizePackageNames(bt *BackendT, pkgs []api.PkgInfo) []api.PkgInfo {
+	for idx, pkg := range pkgs {
+		out := (string)(bt.Backend.NormalizePackageName(api.PkgName(pkg.Name)))
+		if pkgs[idx].Name != out {
+			bt.t.Log("Inconsistently normalized package name in results: ", pkgs[idx].Name, "did not equal", out)
+		}
+		pkgs[idx].Name = out
+	}
+	return pkgs
+}
+
```

Comment:
let's address the normalization stuff in another PR? i wanna have a better view of how much logic this affects and where we have the wrong logic for normalized package names

Reply:
Ope, merged in #183. So, the reason I added this here is to ensure the tests don't break, a subsequent PR should actually _remove_ this, once we're normalized.